.golangci-lint.yml

```diff
@@ -78,10 +78,10 @@ linters:
     - gosimple
     - staticcheck
     - unused
-    #TODO: add revive like in other projects
+    # TODO: add revive like in other projects
 
 linters-settings:
-  #TODO: setup like in other projects
+  # TODO: setup like in other projects
```

Comment:
Do we have a "reminder" to do so? I mean: jira task or something?

Reply:
No, but I will take care of it just after this PR